corehq/apps/case_importer_v1/util.py

```diff
@@ -139,10 +135,16 @@ def _row_values(self, sheet, index):
         return [self._fmt_value(cell) for cell in sheet.row(index)]
 
     def _fmt_value(self, cell):
-        # Explicitly format integers, since xlrd treats all numbers as decimals (adds ".0")
-        if cell.ctype == xlrd.XL_CELL_NUMBER and int(cell.value) == cell.value:
-            return int(cell.value)
-        return cell.value
+        if cell.ctype == xlrd.XL_CELL_NUMBER:
+            if int(cell.value) == cell.value:
```

Comment:
That's both currently the case and desired. If you type "1" into a cell in excel, this library will give you '1.0' as its value. In order to have the string "1" appear in the case property—almost always what's desired and unlikely to present a problem even if you did want a float—we cast to int if that doesn't change the numerical value (i.e. doesn't drop precision after the `.`).